localflavor/generic/forms.py

```diff
@@ -127,3 +130,13 @@ def prepare_value(self, value):
         if value is not None:
             return value.upper()
         return value
+
+
+class PhoneNumberFormFieldMixin(object):
+    def __init__(self):
+        super(PhoneNumberFormFieldMixin, self).__init__()
+        warnings.warn(
```

Comment:
To be consistent with Django, I'd rather if you used `from __future__ import unicode_literals` rather than u prefixes (which are obsolete when dropping support for Python 2).

Django style is:
```
warnings.warn(
    "PhoneNumber form fields are deprecated in favor of the "
    "django-phonenumber-field library.",
    RemovedInLocalflavor20Warning,
)
```